### PR TITLE
add support beqs

### PR DESCRIPTION
**Pull request by avantgardeam**
Hello!

Last week I opened an issue (#25) from my personal account. The issue relates to the support for beqs, and I believe I have addressed it in this pull request.

However, this is my first pull request ever, so I'm not entirely sure if everything is in order. Any feedback on that matter would be great!

Finally, if you think it's good to go, I can start work on some documentation for the new functionality. 

Thanks!

### CODE REVIEW

**Comment by matthewgilbert** [comment]
This looks good, I pushed one commit to add some slight code formatting since I try to follow black code formatting conventions https://black.readthedocs.io/en/stable/

**Comment by matthewgilbert** [comment]
Just wanted to confirm you have ran this and it works as expected? I don't currently have access to a terminal so I'm unable to test the feature.

File: src/blp/blp.py [hunk 1]
```diff
@@ -803,6 +800,44 @@ def collect_to_bdh(self, responses: Iterable) -> Dict[str, pandas.DataFrame]:
...
+            overrides: List of tuples containing the field to override and its value
+            options: key value pairs to to set in request
+
+        Returns: A pandas.DataFrame with columns ['security', eqs_data[0], ...]
+        """
+        query = create_eqs_query(screen_name, screen_type, overrides, options)
+        df = self.query(query, self.parser, self.collect_to_beqs)
+        columns = ["security"] + sorted([col for col in df.columns if col != "security"])
+        df = (
+            df.sort_values("security").reset_index(drop=True).loc[:, columns].drop(["Ticker"], axis=1)
```
Comment (matthewgilbert) [inline src/blp/blp.py:824]: What's the reason for dropping `Ticker`? Is this a redundant piece of information that is returned by the Bloomberg api that is always equal to `security`?

**Comment by avantgardeam** [comment]
> Just wanted to confirm you have ran this and it works as expected?

I have tested a few examples, and everything seems to be functioning well. However, I haven't conducted a systematic test. If you have specific tests you'd like me to run, feel free to share them, and I'll be happy to execute them.

File: src/blp/blp.py (hunk 1 above)
Comment (avantgardeam) [reply src/blp/blp.py:824]: Exactly! However, if you prefer, we could just leave the Ticker information and let the user drop it if he wants to.

Comment (matthewgilbert) [reply src/blp/blp.py:824]: Ya I think that would be more consistent with other approaches in the library, where I try to pass through the Bloomberg response in a fairly unmodified way.

**Comment by avantgardeam** [comment]
Hello, @matthewgilbert . Following your recommendations, we're now retaining the 'Ticker' column instead of dropping it. Furthermore, since EQS presents the metric and rank side-by-side, I've eliminated the portion of the code responsible for alphabetical column sorting.

**Comment by matthewgilbert** [comment]
Great, merged! Thanks for contributing.

**Pull request merged**